packages/gcf-utils/src/gcf-logger.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
```

Comment:
Simply extracted the GCFLogger related code to a separate file here (from gcf-utils.ts). Code remains unchanged